qoqo/tests/integration/operations/single_qubit_gate_operations.rs

```diff
@@ -2776,6 +2776,7 @@ fn test_pyo3_json_schema(operation: SingleQubitGateOperation) {
         let minimum_version: String = match operation {
             SingleQubitGateOperation::GPi(_) => "1.4.0".to_string(),
             SingleQubitGateOperation::GPi2(_) => "1.4.0".to_string(),
+            SingleQubitGateOperation::Identity(_) => "1.7.0".to_string(),
```

Comment:
I'm curious, why didn't this fail before?

Reply:
It did, but it's behind a feature gate